The MUT/LogManager.swift

```diff
@@ -82,20 +77,59 @@ public class logManager {
             //NSLog("[ERROR ]: An error occured while writing to the Log. \(error).")
         }
     }
-    
-    func fatalWrite(logString: String) {
-        createDirectory()
-        let currentTime = generateCurrentTimeStamp()
-        let logURL = libraryURL?.appendingPathComponent("MUT.log")
-        let dateLogString = currentTime + " [FATAL ]: " + logString
-        //NSLog("[INFO  ]: Writing to MUT log file: '\(logString)'.")
-        do {
-            try dateLogString.appendLineToURL(fileURL: logURL!)
+
+    // Warn will write at levels 1 and higher
+    func warnWrite(logString: String) {
+        if logDefaults.integer(forKey: "LogLevel") >= 1 {
+            createDirectory()
+            let currentTime = generateCurrentTimeStamp()
+            let logURL = libraryURL?.appendingPathComponent("MUT.log")
+            let dateLogString = currentTime + " [WARN  ]: " + logString
+            //NSLog("[INFO  ]: Writing to MUT log file: '\(logString)'.")
+            do {
+                try dateLogString.appendLineToURL(fileURL: logURL!)
+            }
+            catch {
+                //NSLog("[ERROR ]: An error occured while writing to the Log. \(error).")
+            }
         }
-        catch {
-            //NSLog("[ERROR ]: An error occured while writing to the Log. \(error).")
+    }
+
+    // Info will write at levels 2 and higher
+    func infoWrite(logString: String) {
+        if logDefaults.integer(forKey: "LogLevel") >= 2 {
+            createDirectory()
+            let currentTime = generateCurrentTimeStamp()
+            let logURL = libraryURL?.appendingPathComponent("MUT.log")
+            let dateLogString = currentTime + " [INFO  ]: " + logString
+            //NSLog("[INFO  ]: Writing to MUT log file: '\(logString)'.")
+            do {
+                try dateLogString.appendLineToURL(fileURL: logURL!)
+            }
+            catch {
+                //NSLog("[ERROR ]: An error occured while writing to the Log. \(error).")
+            }
         }
     }
+    
+    
+    // Verbose will write at levels 2 and higher
+    func verboseWrite(logString: String) {
```

Comment:
There's a lot of repetition here, these functions would be easier to manage if we move the logic a bit:
```
    func verboseWrite(logString: String) {
        writeLog(logString: logString, level: 3, logType: " [DEBUG ]: ")
    }
    
    private func writeLog(logString: String, level: Int, logType: String) {
        if logDefaults.integer(forKey: "LogLevel") >= level {
            createDirectory()
            let currentTime = generateCurrentTimeStamp()
            let logURL = libraryURL?.appendingPathComponent("MUT.log")
            let dateLogString = currentTime +  logType + logString
            //NSLog("[INFO  ]: Writing to MUT log file: '\(logString)'.")
            do {
                try dateLogString.appendLineToURL(fileURL: logURL!)
            }
            catch {
                //NSLog("[ERROR ]: An error occured while writing to the Log. \(error).")
            }
        }
    }
```